src/main/java/com/cqse/teamscaleupload/TeamscaleUpload.java

```diff
@@ -243,17 +305,30 @@ private static void sendRequestForFormat(Input input, String format, List<String
                 .post(requestBody)
                 .build();
 
+        System.out.print("Sending upload for format " + format + "...");
+        sendRequest(input, url, request);
+    }
+
+    private static String sendRequest(Input input, HttpUrl url, Request request) throws IOException {
         OkHttpClient client = OkHttpClientUtils.createClient(input.validateSsl);
 
         try (Response response = client.newCall(request).execute()) {
             handleCommonErrors(response, input);
```

Comment:
If I remember correctly, handleCommonErrors actually handles _all_ errors. Should we rename it to `handleErrors`?